registries/_tag-kind/nav.md

```diff
@@ -0,0 +1,13 @@
+---
+owner: lornajane
+issue:
+description: "Tags with `kind: nav` are used in documentation to group operations into sections"
+layout: default
+---
+
+{% capture summary %}
+Tags with `kind: nav` are used in documentation to group operations into sections.
+In most cases, an operation only has one tag of this tag so it belongs in one section.
```

Comment:
Maybe this should be ?
```suggestion
In most cases, an operation only has one tag of this type so it belongs in one section.
```

Reply:
or "kind", if that's the word we're going with?